packages/config/config/devices/0x0258/neo-ds07z.json

```
@@ -0,0 +1,158 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "NEO-DS07Z",
+	"description": "Door/Window Sensor Plus",
+	"devices": [
+		{
+			"productType": "0x0010",
+			"productId": "0x0702"
+		}
+	],
+	"firmwareVersion": {
+		"min": "2.0",
+		"max": "255.255"
+	},
+	"metadata": {
+		"inclusion": "1. Power Up The Device.\n2. Set Z-Wave Controller Into Inclusion Mode\n3. Press And Hold The Button For 5s Until White Led Lights Is On, Then Release The Button Before Led Turn Off.\nBlue Led Will Blink With 1s Interval Until Inclusion Successful.",
+		"exclusion": "1. Power Up The Device.\n2. Set Z-Wave Controller Into Exclusion Mode\n3. Press And Hold The Button For 5s Until White Led Lights Is On, Then Release The Button Before Led Turn Off.\nBlue Led Will Blink With 0.5sinterval Until Exclusion Successful.",
+		"reset": "1. Power Up The Device.\n2. Press And Hold The Button For 10s Until Pink Led Lights Is On, Then Release The Button Before Led Turn Off."
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Led Indicated",
```

Should probably be
```suggestion
			"label": "LED Indication",
```
or something like that